app/assets/javascripts/admin/admin_rest_api.js

```diff
@@ -548,43 +550,37 @@ export async function getActiveDatasets(): Promise<Array<APIDatasetType>> {
   return datasets;
 }
 
-export async function getDataset(
-  datasetName: string,
-  sharingToken?: string,
-): Promise<APIDatasetType> {
+export function getDataset(datasetName: string, sharingToken?: ?string): Promise<APIDatasetType> {
```

Comment:
I converted some async functions to normal functions returning a promise, since the old `value = await ...; return value` is unnecessarily verbose.